src/log-to-output.js

```diff
@@ -86,13 +86,14 @@ function install(on, filter) {
 }
 
 function browserLaunchHandler(browser = {}, args) {
-  if (!['chrome'].includes(browser.family)) {
+  const isChrome = ['chrome'].includes(browser.family) || browser.name === 'chrome'
```

Comment:
`name` can also be `canary` or `chromium` (this is why the convenient `browser.family` prop was added :smile: )